coalib/output/printers/LogHelper.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import traceback
+
+
+def log_exception(exception, **kwargs):
+    if not isinstance(exception, BaseException):
+        raise TypeError('log_exception can only log derivatives of '
+                        'BaseException.')
```

Comment:
You're right. `traceback.format_exception` is throwing exception if we don't pass the object of exception. :smile: